src/main/models/wallet-setting.js

```diff
@@ -18,7 +18,7 @@ class WalletSetting extends BaseModel {
 			properties: {
 				id: { type: 'integer' },
 				walletId: { type: 'integer' },
-				showDesktopNotifications: { type: 'integer' },
+				sowDesktopNotifications: { type: 'integer' },
```

Comment:
@sk91 I think there is a typo here.

Reply:
@rodrigopavezi should we care about getting this fixed?

This is a regression. Happened long time ago in initial db schema. Bubbled up to new models.
Sqlite does not support column rename support. To rename it now we need to drop the whole table and create from scratch:

Migration will look like this:

```
exports.up = async (knex, Promise) => {
	// rename sowDestopNotifications to showDesktopNotifications
	await knex.schema.renameTable('wallet_settings', 'wallet_settings_bkp');
	await knex.schema.createTable('wallet_settings', table => {
		table.increments('id');
		table
			.integer('walletId')
			.notNullable()
			.references('wallets.id');
		table
			.integer('showDesktopNotifications')
			.notNullable()
			.defaultTo(0);
		table.string('airDropCode');
		table
			.integer('txHistoryLastSyncedBlock')
			.notNull()
			.defaultTo(0);
		table.integer('createdAt').notNullable();
		table.integer('updatedAt');
	});
	let data = await knex('wallet_settings_old').select(
		'id',
		'walletId',
		'sowDesktopNotifications as showDesktopNotifications',
		'airDropCode',
		'txHistoryLastSyncedBlock',
		'createdAt',
		'updatedAt'
	);
	await knex('wallet_settings').insert(data);
};

exports.down = async (knex, Promise) => {};
```